components/automate-ui/src/app/services/app-config/app-config.service.ts

```diff
@@ -0,0 +1,47 @@
+import { HttpClient, HttpBackend } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+interface ConfigTypes {
+  show?: boolean;
+  message?: string;
+  background_color?: string;
+  text_color?: string;
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+
+
+export class AppConfigService {
+
+  public appConfig: ConfigTypes;
+
+  constructor(private handler: HttpBackend) { }
+
+  public loadAppConfig() {
```

Comment:
This is the function that is doing all the heavy lifting.  It is reaching out to `/banner.js` and returning our configuration data